### PR TITLE
Update mysql.md

### PR DESCRIPTION
**Pull request by dragonsahead**
I kept the error "RSA public key is not available client side" on my head for a couple of days... it was because of the updated auth mechanism so I updated that in the documentation

### CODE REVIEW

**Comment by paoliniluis** [comment]
BTW: should this be here or in the troubleshooting guide @jeff-bruemmer ?

File: docs/administration-guide/databases/mysql.md [hunk 1]
```diff
@@ -5,3 +5,6 @@
 Metabase uses the MariaDB connector to connect to MariaDB and MySQL servers. The MariaDB connector does not currently support MySQL 8's default authentication plugin, so in order to connect, you'll need to change the plugin used by the Metabase user to `mysql_native_password`: `ALTER USER 'metabase'@'%' IDENTIFIED WITH mysql_native_password BY 'thepassword';`
 
 If you're still experiencing problems connecting, please refer to the [troubleshooting guide](../../troubleshooting-guide/datawarehouse.html#mysql-unable-to-log-in-with-correct-credentials).
+
+### Raising a MySQL Docker container of MySQL 8+
+If you are spinning up a new MySQL container and you want Metabase to connect to it without having to manually create the user or change the authentication mechanism, or you are facing the `RSA public key is not available client side (option serverRsaPublicKeyFile not set)` error, use the `['--sql_mode=', '--default-authentication-plugin=mysql_native_password']` modifiers when you are running the container. This will ensure that the root user and the user you create keep using the `mysql_native_password` authentication mechanism.
```
Comment (jeff-bruemmer) [inline docs/administration-guide/databases/mysql.md:10]:
```suggestion
### Raising a MySQL Docker container of MySQL 8+

If you are spinning up a new MySQL container, and

 - you want Metabase to connect to the container without having to manually create the user or change the authentication mechanism,
 - or you're facing a `RSA public key is not available client side (option serverRsaPublicKeyFile not set)` error,
 
Use the `['--sql_mode=', '--default-authentication-plugin=mysql_native_password']` modifiers when you run the container, like so:
 
 ```
 Example command here
```

This option will ensure that the root user, and the user you create, keep using the `mysql_native_password` authentication mechanism.
```

Good note. This doesn't belong in the troubleshooting sections, as it doesn't necessarily correct a problem (public key error aside). 

I included a suggestion for formatting to make the paragraph easier to parse, and added a space to include an command so people could see how the modifier is used in context.

**Review by jeff-bruemmer — APPROVED** [review]
Good note, @paoliniluis. Just a small suggestion, which you can take or leave.

**Comment by paoliniluis** [comment]
Thanks @jeff-bruemmer ! I just commited a few examples as well (but I'm not sure about the format (specially the docker-compose one that should be one line under the other. What would be the best option in markdown to put a yml syntax?

**Comment by jeff-bruemmer** [comment]
>  What would be the best option in markdown to put a yml syntax?

I would wrap the code in triple backticks (```), like so:

```
yaml code here
```

(You'll need to click edit on this comment to view the raw markdown.)

**Comment by nemanjaglumac** [comment]
@paoliniluis you might want to push another commit or to rebase so that the CLA Bot gets updated. Either that, or someone from the GH admins team should force-merge this for you :)

**Review by nemanjaglumac — APPROVED** [review]

**Pull request merged**